### ISSUE TITLE
[ICC][Death Knight] Frost Strike scaling with Essence of the Blood Queen

### ISSUE THREAD

**Issue by plznoz** [dalaranwow/dalaran-wow#4728]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Currently Frost Strike does 4x the normal damage with the debuff Essence of the Blood Queen.

**Expected behaviour**: Tell us what should happen instead
Frost Strike should do 2x damage with the debuff Essence of the Blood Queen.

**Steps to reproduce the problem**:

1. Play Frost Death Knight on Blood Queen Lana'Thel
2. Get the bite
3. Do 40-50k main hand Frost Strikes.

**Include proofs for this behaviour**
**On Dalaran 2017-05-28:**
![http://i.imgur.com/yj6ng6w.png](http://i.imgur.com/yj6ng6w.png)
4x the damage after bite. 2x expected.

**On LIVE 2010-09-30:**
Proof it's from 2010-09-30: http://www.worldoflogs.com/guild/us/undermine/mostly-harmless/calendar/9-2010/
http://www.worldoflogs.com/reports/pnm75e79ulb3ktaz/xe/?s=4940&e=5263&x=%28sourceName%3D%22Morgothe%22+and+spell%3D%22Frost+Strike%22%29+or+%28targetName%3D%22Morgothe%22+and+spell%3D%22Essence+of+the+Blood+Queen%22+and+fulltype%3DSPELL_AURA_APPLIED%29
![http://i.imgur.com/1PlXiXj.png](http://i.imgur.com/1PlXiXj.png)

We see that it's not making the damage 4x, but 2x as expected.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Frost Strike: MH **55268** / **66962** OH
Essence of the Blood Queen: initial debuff **71473** , the following debuffs **71525.**
(NB: LIVE had different IDs for the debuff: **71533,** **71531)**

**Include Screenshots from the issue if necessary**

**Comment by plznoz** [comment]
Along with #4575, this should be highly prioritized ASAP.

**Comment by fsironman** [comment]
Along with all the other fucking class bugs which should have been long applied before ICC.

**Comment by ashido96** [comment]
Isn't frost strike double dipping on damage modifiers a known bug? Like it does it on the rune for IC frost strike does 225% instead of 150%.

**Comment by plznoz** [comment]
Sure, it's known, but not reported afaict.
The problem is that this server doesn't seem to apply anything that's "fixed" on the bugtracker.

**Comment by Rushor** [comment]
Game/Unit: Fix double dipping of mechanic effects 

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4728
* commit: a441c36ce617fe691a8dbd4ebbc73a76df6604c5

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by robinsch** [comment]
@Rushor I ported this to PTR.

**Comment by TeridanDW** [comment]
Hello after few tests on PTR the results are:

> Game/Unit: Fix double dipping of mechanic effects

is still an issue as:

Before the bite:
![wowscrnshot_101917_201916](https://user-images.githubusercontent.com/6797611/31784649-31dc820c-b50b-11e7-9c5f-57e56c05c900.jpg)

After the bite:
![wowscrnshot_101917_202052](https://user-images.githubusercontent.com/6797611/31784658-38591974-b50b-11e7-8b1a-be0ee37d6a7f.jpg)

**Comment by TeridanDW** [comment]
Hello after a fast test on PTR:
 - The 4x scaling issue is still present

![image](https://user-images.githubusercontent.com/6797611/35582617-f605267e-05f7-11e8-91fc-4f32f182b8c3.png)
![image](https://user-images.githubusercontent.com/6797611/35582588-e6de85c8-05f7-11e8-808d-9d55da043b74.png)

**Comment by Aridam** [comment]
Is this still an issue?
